### PR TITLE
Pt 159497193 coinbase in vm state

### PR DESCRIPTION
**Pull request by happi**

### CODE REVIEW

File: deployment/DB_VERSION [hunk 1]
```diff
@@ -1 +1,2 @@
-84
+85
+
```
Comment (dincho) [inline deployment/DB_VERSION]: not sure if this extra line will not break the deployment: https://github.com/aeternity/epoch/blob/master/.circleci/config.yml#L158

File: apps/aecontract/test/aecontract_SUITE.erl [hunk 2]
```diff
@@ -70,9 +73,30 @@
...
+    case proplists:get_value(old_beneficiary, Cfg) of
+        undefined ->
+            ok = application:unset_env(aecore, beneficiary);
+        {ok, Old} ->
+            application:set_env(aecore, beneficiary, Old)
+    end,
+    Cfg.
+
+setup_minimal() ->
+    ok = application:set_env(aecore, beneficiary,
```
Comment (lucafavatella) [inline apps/aecontract/test/aecontract_SUITE.erl]: I see https://github.com/aeternity/epoch/blob/e3296af5398432e937965c81239cffe0a81ac47d/apps/aecore/src/aec_conductor.erl#L292

Ok.

File: apps/aecontract/test/aecontract_SUITE.erl [hunk 3]
```diff
@@ -1791,6 +1816,17 @@ sophia_variant_types(_Cfg) ->
     ok.
 
 
+sophia_chain(_Cfg) ->
+    state(aect_test_utils:new_state()),
+    Acc = <<AccId:256>> = ?call(new_account, 1000000),
+    ok = application:set_env(aecore, beneficiary,
```
Comment (lucafavatella) [inline apps/aecontract/test/aecontract_SUITE.erl]: I see end_per_suite cleans this. Ok.

File: apps/aecontract/test/aecontract_SUITE.erl [hunk 4]
```diff
@@ -1791,6 +1816,17 @@ sophia_variant_types(_Cfg) ->
...
+                             aec_base58c:encode(account_pubkey, Acc)),
+    {ok, Acc} = aec_conductor:get_beneficiary(),
+    Ct1       = ?call(create_contract, Acc, chain, {}, #{amount => 10000}),
+    AccId     = ?call(call_contract, Acc, Ct1, miner, word, {}),
```
Comment (lucafavatella) [inline apps/aecontract/test/aecontract_SUITE.erl]: I see AccId bound above so this is assertion. Ok.

**Comment by lucafavatella** [comment]
I see [CI test job](https://circleci.com/gh/aeternity/epoch/28842) failed with:
```
%%% aevm_chain_SUITE ==> all_tests.spend.spend: FAILED
%%% aevm_chain_SUITE ==> {{badmatch,{error,beneficiary_not_configured}},
 [{aect_dispatch,set_env,5,
                 [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_dispatch.erl"},
                  {line,90}]},
  {aect_dispatch,call_AEVM_01_Sophia_01,1,
                 [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_dispatch.erl"},
                  {line,72}]},
  {aect_create_tx,process,6,
                  [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_create_tx.erl"},
                   {line,231}]},
  {aec_trees,apply_txs_on_state_trees,7,
             [{file,"/home/builder/epoch/_build/test/lib/aecore/src/aec_trees.erl"},
              {line,356}]},
  {aec_block_micro_candidate,int_apply_block_txs,5,
                             [{file,"/home/builder/epoch/_build/test/lib/aecore/src/aec_block_micro_candidate.erl"},
                              {line,135}]},
  {aevm_chain_SUITE,sign_and_apply_transaction,3,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,75}]},
  {aevm_chain_SUITE,create_contract,2,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,67}]},
  {aevm_chain_SUITE,setup_chain,0,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,51}]}]}
%%% aevm_chain_SUITE ==> all_tests.contracts.contracts: FAILED
%%% aevm_chain_SUITE ==> {{badmatch,{error,beneficiary_not_configured}},
 [{aect_dispatch,set_env,5,
                 [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_dispatch.erl"},
                  {line,90}]},
  {aect_dispatch,call_AEVM_01_Sophia_01,1,
                 [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_dispatch.erl"},
                  {line,72}]},
  {aect_create_tx,process,6,
                  [{file,"/home/builder/epoch/_build/test/lib/aecontract/src/aect_create_tx.erl"},
                   {line,231}]},
  {aec_trees,apply_txs_on_state_trees,7,
             [{file,"/home/builder/epoch/_build/test/lib/aecore/src/aec_trees.erl"},
              {line,356}]},
  {aec_block_micro_candidate,int_apply_block_txs,5,
                             [{file,"/home/builder/epoch/_build/test/lib/aecore/src/aec_block_micro_candidate.erl"},
                              {line,135}]},
  {aevm_chain_SUITE,sign_and_apply_transaction,3,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,75}]},
  {aevm_chain_SUITE,create_contract,2,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,67}]},
  {aevm_chain_SUITE,setup_chain,0,
                    [{file,"/home/builder/epoch/_build/test/extras/test/aevm_chain_SUITE.erl"},
                     {line,51}]}]}
```

**Comment by lucafavatella** [comment]
> I see [CI test job](https://circleci.com/gh/aeternity/epoch/28842) failed with:

[Past CI test job](https://circleci.com/gh/aeternity/epoch/28242) passed - same code (changed only new line in DB_VERSION file).

**Comment by lucafavatella** [comment]
Rebased, fixed conflict on release notes and DB folder, force-pushed.

**Pull request closed without merging**